app/controllers/report_controller/reports.rb

```diff
@@ -123,7 +123,7 @@ def get_all_reps(nodeid = nil)
         @settings.store_path(:perpage, :list, @items_per_page) # Set the per page setting for this gtl type
       end
 
-      @sortcol = session["#{x_active_tree}_sortcol".to_sym].nil? ? 0 : session["#{x_active_tree}_sortcol".to_sym].to_i
+      @sortcol = session["#{x_active_tree}_sortcol".to_sym].nil? ? 2 : session["#{x_active_tree}_sortcol".to_sym].to_i
```

Comment:
Recommend moving the number into a constant, so it's not a magic number.

Reply:
Maybe in report_controller something like `ReportController::DEFAULT_SORT_COL = 2`, then up above in ApplicationController you can reference it as well